tests/dns/dns-rcode/test.yaml

```diff
@@ -0,0 +1,36 @@
+requires:
+  min-version: 8
+
+pcap: input.pcap
```

Comment:
The `pcap` option for `test.yaml` is only required when the test itself doesn't have a pcap file (either because it doesn't require one, or because it uses a pcap from a pre-existing test). For the latter case - which is the case for your tests, the usage is to pass the path for the pcap in the existing test. (see https://github.com/OISF/suricata-verify/blob/master/tests/exception-policy-midstream-02/test.yaml#L1 for example :) )

And, once you have that, you don't need to include the pcap in your newly created test.